pkg/apiserver/domain/service/definition.go

```diff
@@ -109,6 +110,13 @@ func (d *definitionServiceImpl) listDefinitions(ctx context.Context, list *unstr
 			},
 		},
 	}
+	if ops.FilterOrigin != "" {
+		matchLabels.MatchExpressions = append(matchLabels.MatchExpressions, metav1.LabelSelectorRequirement{
+			Key:      types.LabelDefinitionOrigin,
+			Operator: metav1.LabelSelectorOpNotIn,
```

Comment:
LabelSelectorOpNotIn ?

Reply:
There's no `or` in label selectors, so the input here is `FilterOrigin` which need to be filtered.

Reply:
This is a little weird. If users want to query the Application Workflow step definitions, you could filter the Workflow Run in the implementation. The logic is transparent to the API user.